packages/cashier/src/components/transfer-confirm/transfer-confirm.tsx

```diff
@@ -43,7 +43,7 @@ const Row = ({ item_key, label, value }: TRowProps) => (
         {Array.isArray(value) ? (
             <div className='transfer-confirm__row-value'>
                 {value.map(v => (
-                    <Text as='div' key={v} size='xs' weight='bold' align='right'>
+                    <Text as='div' className='transfer-confirm__row-details' key={v} size='xs' weight='bold'>
```

Comment:
hey @lubega-deriv could we add `--text-align-left: end;` variable to `themes.scss` and use align="end" prop in Text component?

Reply:
Updated